### ISSUE TITLE
[FEATURE]: Information on Embedathon homepage.

### ISSUE THREAD

**Issue by anirudhprabhakaran3** [IEEE-NITK/corpus#41]
### What is the feature you would like to see?

Add the following text to the embedathon homepage.

Those who are taking IEEE/Renewing membership can add CAS membership for 2024 for FREE. Please encourage more students to get the benefit.

Please note its only applicable to those who have never been CAS Member before.
https://cas.ieeebangalore.org/guide-to-join-ieee-cas/

![cas_membership](https://github.com/IEEE-NITK/corpus/assets/76864239/3c8540b4-ac9e-4c7a-85ef-b4314e1a9765)

### How important do you think this feature is?

Medium

**Comment by nishant-nayak** [comment]
@anirudhprabhakaran3 shouldn't this be a part of the [Embedathon-Website](https://github.com/IEEE-NITK/Embedathon-Website) repo?

**Comment by anirudhprabhakaran3** [comment]
Migrating that project here right?

**Comment by nishant-nayak** [comment]
@anirudhprabhakaran3 migrate once the public facing pages of Corpus are ready and the Embedathon Website for 2023 is fully ready. Until then, I'd recommend developing the Embedathon Website as a separate project.

I don't want Embedathon folks waiting on us to finish migrating the public pages before they can get their registrations and all underway.

**Comment by anirudhprabhakaran3** [comment]
yeah so I'm getting vignaraj and a couple of juniors to start migrating the static pages from now itself, so that by the time we need to get this live we'll have all the content ready. that's fine right?

**Comment by nishant-nayak** [comment]
That's fine. *If* all goes well, we can have Embedathon as part of this project itself. To be safer, I would still recommend developing the Embedathon Website separately, maybe creating provisions for easy integration with Corpus.

**Comment by anirudhprabhakaran3** [comment]
I think we can do it, since even if we don't get all static pages done by then, we'll have to get a new container for Embedathon and a domain. If that's the case, we can just disable some urls and deploy just the embedathon app right? That's why developing it in that different repo seems a bit redundant, imo.

Also will be running behind the others to get static work asap. should not be that difficult tbh.

**Comment by nishant-nayak** [comment]
Alright. Get whoever is working on Embedathon to migrate the Embedathon app into this repo and make a new PR for it.

I'll also be working on migrating static pages. If there is anyone else working on new pages, make sure they have an issue assigned to them for the page. #37 and #38 are good starting points.

**Comment by anirudhprabhakaran3** [comment]
I only will start working on the Embedathon app for now. Will ask Vignaraj if he wants to join or if he'll be part of the migration force too.

Also, scaffolding Embedathon from scratch is a good idea? What do you think? (from scratch I mean copy pasting relevant pieces of code from the old repo, not just copy pasting entire files and all)

**Comment by nishant-nayak** [comment]
Taking this discussion onto Discord.
